tasks/tls_signed.yml

```diff
@@ -1,32 +1,53 @@
 ---
-- name: "[SIGNED TLS] - Define signed certificate path"
+- name: "[SIGNED TLS] - Define signed certificate-file with abolute path"
   ansible.builtin.set_fact:
-    nextcloud_tls_cert_file: "{{ nextcloud_tls_cert | default(\"/etc/ssl/\" + nextcloud_instance_name + \".crt\") }}"
+    nextcloud_tls_cert_file: "{{ nextcloud_tls_cert | default(cert_path + nextcloud_instance_name + \".crt\") }}"
```

Comment:
While checking changes in README I've noticed that: `nextcloud_tls_cert` is mentioned in README only in `installed` option. So maybe there is documentation missing or it's a typo here. Could you check @marioqxx?

Reply:
In this line there is:
    nextcloud_tls_cert_file: "{{ **nextcloud_tls_cert** | default(cert_path + nextcloud_instance_name + \".crt\") }}"
which is not defined in **signed** section in README.

Reply:
Hi @Wiktor2200, I have changed this. Actually this was code I did not touch, except for replacing with "cert_path". However, for consistency I decided to add "_file" to the new introduced "nextcloud_tls_chain_file", because this variable shall refer to a path + filename. In editing the README I (unknowingly) assumed the same were true for the existing "nextcloud_tls_cert" and "nextcloud_tls_cert_key" variables. I have now changes this in "tls:signed.yml". README shall now be correct as well as "default/main.yml".